### DISCUSSION TITLE
YAML code for specific sample graph

### DISCUSSION THREAD

**Discussion by Qhilm** [dbuezas/lovelace-plotly-graph-card#326] [Show and tell]
https://user-images.githubusercontent.com/777196/198649220-14af3cf2-8948-4174-8138-b669dce5319e.png

How do I do this? I can do the mean value, but I can't figure out how to create this "envelope" lines, above and below the mean.

**Comment by dbuezas** [comment]
See this yaml: https://github.com/dbuezas/lovelace-plotly-graph-card/discussions/75#discussioncomment-3979606

Ignore the issues he had, that bug is fixed
